penaltymodel_lp/test_generation.py

```diff
@@ -21,7 +21,7 @@
 #TODO: add tests on satisfying min_gap. Currently, we're always checking that gap > 0, and passive
 # check that gap >= default 2.
 class TestPenaltyModelLinearProgramming(unittest.TestCase):
-    def verify_gate_bqm(self, bqm, nodes, get_gate_output, ground_energy=0, min_gap=2):
+    def verify_gate_bqm(self, bqm, nodes, get_gate_output, ground_energy=0, min_gap=2, precision=7):
```

Comment:
Totally optional, but you might consider naming this `places` or `ndigits` to keep it consistent with `assertAlmostEqual` or `round` respectively.